### PR TITLE
Rename to simpla-paths, inline HTML dist

### PR DESCRIPTION
**Pull request by madeleineostoja**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,18 +2,15 @@ sudo: true
...
+- '6'
 before_script:
-- npm install bower
+- npm install -g bower web-component-tester
 - bower install
-script: wct --plugin sauce
-cache:
-  directories:
-  - node_modules
+script: xvfb-run wct
```
Comment (bedeoverend) [inline .travis.yml:9]: @seaneking why the switch from sauce to xvfb?

**Comment by bedeoverend** [comment]
Just made a couple of tweaks - hope that's ok. There was a bug in the gulpfile and just a few files missed in the renaming.

Other than the comment I've made above, all good 👍

File: .travis.yml (hunk 1 above)
Comment (madeleineostoja) [reply .travis.yml:9]: Copying Polymer components, seems to run faster?

**Comment by madeleineostoja** [comment]
Looooks good, I'll merge. Feel free to change xfb thing if you want

File: .travis.yml (hunk 1 above)
Comment (bedeoverend) [reply .travis.yml:9]: Hmm...it just means I don't think it'll run on sauce, which means it'll only test the browsers that's on the travis VM - just Chrome I think - but I'll have a dig into it, particularly if the Polymer guys aren't using sauce.

Comment (madeleineostoja) [reply .travis.yml:9]: Sauce is broken for us, if you haven't noticed. Don't know if they changed their plans or what, but always says we're out of minutes

Comment (bedeoverend) [reply .travis.yml:9]: Ha, no I hadn't. Righto - I'll have a squiz see what's going on

**Pull request merged**